intro-js.md

```diff
@@ -0,0 +1,222 @@
+
+So, getting back to classes. 
+
+Example class
+```javascript 
+class Shape {
+    constructor(param1, param2) {
+        
+    }
+}
+```
+
+The constructor method is what is used to create an instance of the class Shape above. 
+
+You can only have one constructor method per class. 
+
+The parameters for the constructor [param1, param2] are also optional. You could just as easily define the class like below. 
+
+```javascript
+class Shape {
+    constructor() {
+        
+    }
+}
+```
+
+The params represent the properties you want to set on your object. 
+
+### Functions are first class objects
+
+This one is a little hard to wrap your heads around so we won't go into too much detail, but it is one of the most fundamental aspects of JavaScript. 
+
+It is what makes JavaScript such a powerful language. 
+
+Basically, what this means is that functions can be constructed during execution, they can be passed around as variables or parameters to other functions and you can also return a function as the value of another function (what is referred to as currying). 
+
+Confused? That's okay. If you wish to, you can read more about it [here](http://timmknight.github.io/2015/first-class-functions-javascript/)
+
+There are a lot of other reasons, but these will be introduced as the meetings move forward. 
+
+## Basic data types in JavaScript
+Some of the ECMAScript standard datatypes:
+* Boolean 
+* Number
+* String
+
+# JSON
+
+JavaScript Object Notation (JSON for short) is JavaScript object syntax. It is the primary form of information interchange on the web. 
+
+Here is a basic JSON object. 
+
+```javascript
+var jsonObj = {
+    "name": "Zaid Humayun", 
+    "age": "21", 
+    "city": "West Lafayette"
+};
+```
+
+When you have the above JSON object, getting the values for specific key value pairs is simple using the '.' operator. 
+
+To get the name from the above object, you would simply do
+
+```javascript
+    var name = jsonObj.name; //Zaid Humayun
+```
+
+or you could also use square brackets to denote what property on the object you are trying to access
+
+```javascript
+    var name = jsonObj["name"]; //Zaid Humayun
+```
+
+JSON also contains two methods called JSON.parse() and JSON.stringify() to convert between a string and JSON object. 
+
+Read more about them [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON)
+
+# Booleans in JavaScript
+
+JavaScript has two basic boolean values: true and false. 
+
+Setting some value up as a boolean is fairly simple
+
+```javascript
+var a = true; //a is the true boolean value
+var b = false; //b is the false boolean value
+
+//Using these values
+if(a) {
+    //do something here
+}
+if(!b) {
+    //do something else here
+}
+```
+
+But there's something weird about booleans in JS too. Who could have guessed right? 
+
+JavaScript has something called *truthy* and *falsey* values. 
```

Comment:
I wouldn't have thought about adding this, but I think truthy and falsey values are great to bring up earlier in teaching javascript like this. Nice thinking!